js/src/components/google-combo-account-card/connected-google-combo-account-card.js

```diff
@@ -66,22 +79,57 @@ const ConnectedGoogleComboAccountCard = () => {
 		upsertAccount();
 	}, [ finalizeAdsAccountCreation, upsertAdsAccount, invalidateResolution ] );
 
+	useEffect( () => {
+		setShowConversionMeasurementNotice(
+			( googleAdsAccount?.status ===
+				GOOGLE_ADS_ACCOUNT_STATUS.CONNECTED ||
+				googleAdsAccount?.step === 'link_merchant' ) &&
+				! shouldClaimGoogleAdsAccount
+		);
+	}, [ googleAdsAccount, shouldClaimGoogleAdsAccount ] );
```

Comment:
Related to https://github.com/woocommerce/google-listings-and-ads/pull/2644#discussion_r1833445209 and https://github.com/woocommerce/google-listings-and-ads/pull/2644#discussion_r1833997070.

> There is not an explicit requirement for this, but the design doc implies that after all accounts finish connecting and the combo card collapses, that this would no longer be shown.

> Also I think it's safe to say that whenever the "Cancel" button is clicked, we can hide the notice.

Although I still don't think the design document implies this logic (Ref: https://github.com/woocommerce/google-listings-and-ads/pull/2644#discussion_r1833656584), if it's determined that the quoted logic is to be implemented, then the current implementation doesn't seem to satisfy it.

https://github.com/user-attachments/assets/3ef71ac6-fcd0-498b-95dc-7373e129ab94

In the video, there are two issues:

1. If the initial state is that all accounts have finished connecting, the collapsed card still shows the notice.
2. When all accounts have finished connecting and the card is re-expanded, the notice is not shown.

Could it be clarified which logic is intended to be adopted?

Reply:
This was removed in #2644 but those upstream changes have not yet been merged back to this branch. I plan to handle this as part of resolving merge conflicts before asking for another review.